packages/ERTP/src/purse.js

```diff
@@ -45,17 +42,14 @@ export const preparePurseKind = (
   //   that created depositFacet as needed. But this approach ensures a constant
   //   identity for the facet and exercises the multi-faceted object style.
   const { depositInternal, withdrawInternal } = purseMethods;
-  const makePurseKit = prepareExoClassKit(
-    issuerBaggage,
+  const makePurseKit = issuerZone.exoClassKit(
     `${name} Purse`,
     PurseIKit,
     () => {
       const currentBalance = AmountMath.makeEmpty(brand, assetKind);
 
       /** @type {SetStore<Payment>} */
-      const recoverySet = makeScalarBigSetStore('recovery set', {
-        durable: true,
-      });
+      const recoverySet = issuerZone.detached().setStore('recovery set');
```

Comment:
What does `detached()` do? Is there any documentation?

Reply:
`detached()` is how you ask that a new one always be made, even if it is a durable zone. Without `detach()`, `issuerZone.setStore('recovery set')` would reuse a `'recovery set'` entry in `issuerZone` if there is one, and register the newly made one there otherwise. IOW, `detached()` is the new way to say `make*SetStore` rather than `provide*SetStore`.

@michaelfig , I leave the documentation question to you. Thanks.